monitoring/pagerduty.py

```diff
@@ -203,7 +203,7 @@ def create(module, name, user, passwd, token, requester_id, service, hours, minu
 
     data = json.dumps(request_data)
     response, info = fetch_url(module, url, data=data, headers=headers, method='POST')
-    if info['status'] != 200:
+    if info['status'] != 201:
```

Comment:
Since this isn't a documented feature in pagerduty, perhaps it's worth using
```
if info['status'] != 200 and info['status'] != 201:
   ...
```

In case they switch back to 200, which is still a valid success code